app/models/profile.rb

```diff
@@ -24,6 +24,8 @@ class Profile < ApplicationRecord
 
   # NOTE: @citizen428 This is a temporary mapping so we don't break DEV during
   # profile migration/generalization work.
+  # TODO: @msarit Should I remove brand_colors from this mapping? Now that we're
```

Comment:
@citizen428 Please note 🙏🏾

Reply:
Yes, you can remove it, it will no longer be necessary. Thank you!

Reply:
👍🏾